requirements.txt

```diff
@@ -1,21 +1,5 @@
-attrs==21.4.0
-bcrypt==3.2.0
-certifi==2021.10.8
 cffi==1.15.0
-charset-normalizer==2.0.12
-cryptography==36.0.2
-idna==3.3
-jsonschema==4.4.0
-monotonic==1.6
-more-executors==2.10.1
--e git+ssh://git@github.com/jalam453/pubtools-content-gateway.git@3c07c7c9e28ae18a28cdc68aecbaeaeff31df7fc#egg=pubtools_content_gateway
-pushcollector==1.3.0
 pycparser==2.21
-PyNaCl==1.5.0
-pyrsistent==0.18.1
-python-qpid-proton==0.33.0
-python2==1.2
-PyYAML==6.0
-requests==2.27.1
 six==1.16.0
```

Comment:
Is there a particular reason for pinning specific versions?

Reply:
Nop! that's just got created by pip freeze > requirements.txt command. Will update it later. For now, we are not using this file.